src/main/webapp/index.html

```diff
@@ -30,9 +35,9 @@ <h1>Census Viz</h1>
           </select>
           <input type="list" list="location" name="location" pattern="every US state|California|New Jersey">
             <datalist id="location">
-              <option data-value="state" value="every US state">
-              <option data-value="county" value="California">
-              <option data-value="county" value="New Jersey">
+              <option data-value="state" value="every US state"></option>
```

Comment:
Nit: I think the convention for datalist is to not close option tags, eg they don't here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist

Reply:
Thanks, will have to figure out how let the linter allow this